package.json

```diff
@@ -116,7 +116,7 @@
     "rc-field-form": "^0.0.0-alpha.17",
     "rc-form": "^2.4.5",
     "rc-input-number": "~4.5.0",
-    "rc-mentions": "~0.4.0",
+    "rc-mentions": "^1.0.0-alpha.0",
```

Comment:
Use `~` instead of `^`, antd alway lock with rc minor version with each minor version.

Reply:
OK，but `rc-form` or `rc-field-form` need change?

Reply:
Yes. Should be `~` also and need check if `rc-form` still needed or can be removed.